integreat_cms/cms/migrations/0044_alter_user_options.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 3.2.16 on 2022-11-06 10:53
+
+from django.core.management.sql import emit_post_migrate_signal
+from django.db import migrations
+
+
```

Comment:
```suggestion
from ..constants import roles


```
Sorry, forgot this import

Reply:
No problem, I added it :)